internal/pkg/sylog/sylog_dummy.go

```diff
@@ -12,6 +12,8 @@ import (
 	"io"
 	"io/ioutil"
 	"os"
+
```

Comment:
I'm wondering if it's required to be included in `sylog_dummy` to avoid the extra dependency ?

Reply:
By using `(golog.Logger)(sylog.DebugLogger{})` we could avoid the extra dependency in `sylog` and end with something like in the files :

In `sylog_dummy.go` :

```
type DebugLogger struct {}

// Output a log message via sylog.Debugf
func (DebugLogger) Log(v ...interface{}) {}

// Output a formatted log message via sylog.Debugf
func (DebugLogger) Logf(format string, v ...interface{}) {}
```

In `sylog.go` :

```
type DebugLogger struct {}

// Output a log message via sylog.Debugf
func (DebugLogger) Log(v ...interface{}) {
	writef(os.Stderr, debug, "%s", fmt.Sprint(v...))
}

// Output a formatted log message via sylog.Debugf
func (DebugLogger) Logf(format string, v ...interface{}) {
	writef(os.Stderr, debug, format, v...)
}
```